### PR TITLE
Updating linux-rpm-build documentation

### PR DESCRIPTION
**Pull request by PaurushGarg**
**Description:** <Describe what has changed.>
Follow-up PR of: https://github.com/aws-otel/aws-otel.github.io/pull/508

**Link to tracking Issue:** <Issue number if applicable>
https://github.com/aws-otel/aws-otel.github.io/issues/475

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/developers/linux-rpm-demo.md [hunk 1]
```diff
@@ -1,76 +1,11 @@
-### Run AWSOTelCollector Beta on Amazon Linux 2
+### Run ADOT Collector on Amazon Linux 2
+See [AWS Distro for OpenTelemetry documentation](https://aws-otel.github.io/docs/setup/build-collector-as-rpm) to run ADOT Collector on Amazon Linux 2 EC2 host.
```
Comment (bryan-aguilar) [inline docs/developers/linux-rpm-demo.md:2]: This linked documentation appears to be missing instructions on how to verify the package integrity.

**Review by bryan-aguilar — CHANGES_REQUESTED** [review]

File: docs/developers/linux-rpm-demo.md (hunk 1 above)
Comment (rapphil) [reply docs/developers/linux-rpm-demo.md:2]: That link is pointing to instructions of how to build the collector as RPM. does it make sense to check integrity for something that you just build yourself?
https://www.redhat.com/sysadmin/rpm-gpg-verify-packages

However you brought an interesting point. Do we have instructions on how to install from EC2 using the release buckets?

Comment (rapphil) [reply docs/developers/linux-rpm-demo.md:2]: We do but they point for docs in github https://aws-otel.github.io/docs/getting-started/collector -> https://github.com/aws-observability/aws-otel-collector/blob/main/docs/developers/linux-rpm-demo.md

Comment (PaurushGarg) [reply docs/developers/linux-rpm-demo.md:2]: I have now added an alternative step in step 1 - to download rpm package from S3 and added the option to check for package integrity there. Please check this PR: https://github.com/aws-otel/aws-otel.github.io/pull/508. 
Thanks!

**Review by rapphil — APPROVED** [review]

**Pull request merged**